CHANGELOG.md

```diff
@@ -5,6 +5,8 @@
 * [ENHANCEMENT]
 * [BUGFIX]
 
+* [CHANGE] Default GOMAXPROCS to 1
```

Comment:
Can we somehow explain why we do this for readers? So they don't increase it without knowing

Reply:
Sure.